### PR TITLE
Strip Repository path from logs

### LINKED ISSUE #13: strip travis build paths from log
Maybe strip the travis paths from the log outputs like 
`
PROBLEM: Invalid xml found. /home/travis/build/xbmc/repo-plugins/plugin.video.reddit_viewer/resources/settings.xml`

### PR DESCRIPTION
**Pull request by mzfr**
Fixes #13 

Before this log output used to look like:
```
PROBLEM: Invalid xml found. /home/travis/build/xbmc/repo-plugins/
         plugin.video.reddit_viewer/resources/settings.xml
```

After applying these changes log output will look like:
```
PROBLEM: Invalid xml found. /xbmc/repo-plugins/
         plugin.video.reddit_viewer/resources/settings.xml
```

### CODE REVIEW

**Comment by Rechi** [comment]
IMO `/xbmc/repo-plugins/` should also be removed from output

**Comment by mzfr** [comment]
@Rechi I have made the changes you suggested.

File: check_addon.py [hunk 1]
```diff
@@ -103,6 +107,7 @@ def _check_for_invalid_xml_files(error_counter, file_index):
                 # Just try if we can successfully parse it
                 xml.etree.ElementTree.parse(xml_path)
             except xml.etree.ElementTree.ParseError:
+                xml_path = relative_path(xml_path)
```
Comment (Rechi) [inline check_addon.py]: variable shouldn't be overwritten, either create a new variable, or do the `relative_path` function call directly in the log call

**Comment by Rechi** [comment]
@mzfr please squash your changes into a single commit.

**Comment by mzfr** [comment]
@Rechi I have made the changes and have squashed the commits

**Comment by Rechi** [comment]
Tried it now, looks good
Only thing I would change is currently it outputs `/<ADDON-ID>/<FILE>`, but I don't like the slash at the beginning.

**Comment by Razzeee** [comment]
@Rechi 
to be clear do you say you want no slash at all or `./` or `../`

**Comment by Rechi** [comment]
no slash at the beginning (as it would look like an absolute path)
`./` is also okay for me, if you prefer that

**Comment by Razzeee** [comment]
I would in deed prefer `./` as it communicates, that we maybe omited something

**Comment by mzfr** [comment]
@Razzeee  @Rechi I have made the changes suggested.

**Comment by Razzeee** [comment]
Thanks @mzfr

**Pull request merged**